provider/clickhouse.go

```diff
@@ -269,8 +274,12 @@ func (table *clickhouseOfflineTable) Write(rec ResourceRecord) error {
 		return err
 	}
 	insertQuery := table.query.writeInserts(tb)
-	_, err := table.db.Exec(insertQuery, rec.Entity, rec.Value, rec.TS)
-	return err
+	if _, err := table.db.Exec(insertQuery, rec.Entity, rec.Value, rec.TS); err != nil {
+		wrapped := fferr.NewResourceExecutionError(pt.ClickHouseOffline.String(), rec.Entity, "", fferr.ENTITY, err)
```

Comment:
The name variant should be the name variant of the table right?

Reply:
The feature variant is serialized into the table name at this point, so those aren't accessible.